client/my-sites/subscribers/main.tsx

```diff
@@ -1,19 +1,306 @@
+		status: 'inactive',
+		subscription: 'Free Tier',
+		openRate: '70%',
+		date_subscribed: '2022-01-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 13,
+		subscription_id: 13,
+		display_name: 'Beatriz Silva',
+		email_address: 'beasilva@gmail.com',
+		status: 'active',
+		subscription: 'Annual (US$ 50)',
+		openRate: '90%',
+		date_subscribed: '2022-02-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 14,
+		subscription_id: 14,
+		display_name: 'Antonio Romero',
+		email_address: 'antromero@gmail.com',
+		status: 'active',
+		subscription: 'Monthly (US$ 4.99)',
+		openRate: '95%',
+		date_subscribed: '2022-03-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 15,
+		subscription_id: 15,
+		display_name: 'María Guzmán',
+		email_address: 'mariaguzman@gmail.com',
+		status: 'inactive',
+		subscription: 'Free Tier',
+		openRate: '60%',
+		date_subscribed: '2022-04-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 16,
+		subscription_id: 16,
+		display_name: 'Javier Salazar',
+		email_address: 'javisalazar@gmail.com',
+		status: 'active',
+		subscription: 'Annual (US$ 50)',
+		openRate: '100%',
+		date_subscribed: '2022-05-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 17,
+		subscription_id: 17,
+		display_name: 'Laura Mendoza',
+		email_address: 'lauramendoza@gmail.com',
+		status: 'active',
+		subscription: 'Monthly (US$ 4.99)',
+		openRate: '85%',
+		date_subscribed: '2022-06-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 18,
+		subscription_id: 18,
+		display_name: 'Pablo Aguirre',
+		email_address: 'pabloaguirre@gmail.com',
+		status: 'inactive',
+		subscription: 'Free Tier',
+		openRate: '80%',
+		date_subscribed: '2022-07-01T00:00:00.000Z',
+		avatar: '',
+	},
+];
 
 export const Subscribers = () => {
 	const isSubscribersPageEnabled = config.isEnabled( 'subscribers-page' );
+	const locale = useLocale();
+
+	const navigationItems: Item[] = [
+		{
+			label: 'Subscribers',
+			href: `/subscribers`,
+			helpBubble: (
+				<span>
+					{ translate(
+						'Add subscribers to your site and send them a free or paid {{link}}newsletter{{/link}}.',
+						{
+							components: {
+								link: (
+									<a
+										href="https://wordpress.com/support/launch-a-newsletter/#about-your-subscribers"
+										target="blank"
+									/>
+								),
+							},
+						}
+					) }
+				</span>
+			),
+		},
+	];
 
 	if ( ! isSubscribersPageEnabled ) {
 		return null;
 	}
 
 	return (
-		<Main>
+		<>
 			<DocumentHead title={ translate( 'Subscribers' ) } />
-			Subscribers
-		</Main>
+			<div className="main is-wide-layout">
+				<FixedNavigationHeader navigationItems={ navigationItems }></FixedNavigationHeader>
+				<div className="header-count">
+					<span className="title">{ translate( 'Total' ) }</span>{ ' ' }
+					<span className="subscriber-count">{ subscribers && subscribers.length }</span>
+				</div>
+				<table className="subscriber-list">
+					<tbody>
+						<tr className="table-header">
```

Comment:
[EsLint Calypso rules](https://github.com/Automattic/wp-calypso/blob/trunk/packages/eslint-plugin-wpcalypso/docs/rules/jsx-classname-namespace.md) recommends to use BEM naming conventions on classNames. You can check the rules here: https://en.bem.info/methodology/naming-convention/. In this case all the classNames nested to subscriber-list would be updated to include the namespace prefix. Example: `table-header` => `subscriber-list__ table-header`

Reply:
I'll change them